evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -1976,14 +1976,19 @@ def test_score_error_when_custom_objective_not_instantiated(problem_type, logist
         pipeline.score(X, y, objectives=[CostBenefitMatrix(1, 1, -1, -1), "F1"])
 
 
-def test_binary_pipeline_string_target_thresholding(make_data_type, logistic_regression_binary_pipeline_class, X_y_binary):
+def test_binary_pipeline_string_target_thresholding(make_data_type, time_series_binary_classification_pipeline_class,
+                                                    logistic_regression_binary_pipeline_class,
+                                                    X_y_binary):
     X, y = X_y_binary
     X = make_data_type('ww', X)
     y = make_data_type('ww', pd.Series([f"String value {i}" for i in y]))
     objective = get_objective("F1", return_instance=True)
-    pipeline = logistic_regression_binary_pipeline_class(parameters={"Logistic Regression Classifier": {"n_jobs": 1}})
-    pipeline.fit(X, y)
-    assert pipeline.threshold is None
-    pred_proba = pipeline.predict_proba(X, y).iloc[:, 1]
-    pipeline.optimize_threshold(X, y, pred_proba, objective)
-    assert pipeline.threshold is not None
+    for pipeline_class in [time_series_binary_classification_pipeline_class,
+                           logistic_regression_binary_pipeline_class]:
```

Comment:
Might be a nit, but it'll look better in test failure, perhaps paratetrize this one?

Reply:
Good call! I just got lazy cause you can't parametrize over fixtures lol